gradle/published-java-module.gradle

```diff
@@ -87,9 +87,6 @@ javadoc {
 	final int currentYear = new GregorianCalendar().get( Calendar.YEAR )
 
 	configure( options ) {
-		docletpath = configurations.asciidoclet.files.asType(List)
-		doclet = 'org.asciidoctor.Asciidoclet'
-
```

Comment:
This was discussed on list: we are not sure we have Asciidoctor Javadoc content in 5.3 and it breaks all the HTML Javadoc we have. So better remove it in 5.3.